PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+#### Story / Bug id:
+123 [link to the issue]
+
+#### Description:
+Describe changes made in your pull request here
+
+#### Migrations:
+N/A
+
+#### Rollback:
```

Comment:
Isn't it too specific? As I understand rollback means reversed migration (does it?), in which case we would hardly ever use that field and even if we do, "Migrations" still seems to be the likely place I would mention it

Reply:
Defaultly it would be N/A (or the previous version mentioned), but I would stick with that as a good practice. Nevertheless could be removed. ;-)